docker/centos/Dockerfile

```diff
@@ -1,28 +0,0 @@
-FROM centos:latest
```

Comment:
This is sad, since it was written by an external contributor (#9280).

Reply:
yeah, we should put them somewhere else https://github.com/paritytech/scripts